ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -280,6 +280,52 @@ def test_clear_foreign_config_fail(self, mock_validate_job_queue,
             exception.DracOperationError, drac_raid.clear_foreign_config,
             self.node, 'RAID.Integrated.1-1')
 
+    @mock.patch.object(drac_job, 'validate_job_queue', spec_set=True,
+                       autospec=True)
+    def test_change_physical_disk_state(self,
+                                        mock_validate_job_queue,
+                                        mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+        drac_raid.change_physical_disk_state(
+            self.node,
+            mode="JBOD")
+
+        mock_client.change_physical_disk_state.assert_called_once_with(
+            "JBOD")
+
+    @mock.patch.object(drac_raid, 'change_physical_disk_state', spec_set=True,
+                       autospec=True)
+    @mock.patch.object(drac_raid, 'commit_config', spec_set=True,
+                       autospec=True)
+    def test__change_physical_disk_conversion(self,
+                                              mock_commit_config,
+                                              mock_change_physical_disk_state,
+                                              mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+        mock_commit_config.return_value = '42'
+        mock_change_physical_disk_state.return_value = {
+            'is_reboot_required': True,
+            'conversion_results': {
+                'RAID.Integrated.1-1': {'is_reboot_required': 'optional',
+                                        'is_commit_required': True}},
+                'commit_required_ids': ['RAID.Integrated.1-1']}
+
+        drac_raid._change_physical_disk_conversion(
```

Comment:
Probably should check the return value and assert that it is the right value.

Reply:
The only value drac_raid._change_physical_disk_conversion can return is `'clean wait'` so updated raid.py accordingly and assert whether return value of above line `clean_wait` or not.